csp/impl/pandas_perspective.py

```diff
@@ -176,12 +180,28 @@ def __init__(
                 schema[col] = series.dtype.subtype
             else:
                 schema[col] = static_schema[col]
+        if _PERSPECTIVE_3:
+            perspective_type_map = {
```

Comment:
This is a duplicate of the mapping/schema in `dataframe.py`, we can also stick this in `perspective_common.py`.

Reply:
Yes, moved